packages/ai/src/ai/common/models/audio/kokoro_loader.py

```
@@ -0,0 +1,233 @@
+                    with wave.open(wav_path, 'wb') as wavf:
+                        wavf.setnchannels(1)
+                        wavf.setsampwidth(2)
+                        wavf.setframerate(24000)
+                        wavf.writeframes((audio_arr * 32767).astype(np.int16).tobytes())
+                    with open(wav_path, 'rb') as f:
+                        raw = f.read()
+                finally:
+                    try:
+                        os.remove(wav_path)
+                    except OSError:
+                        pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)